lib/mix/tasks/phx.gen.context.ex

```diff
@@ -99,7 +99,10 @@ defmodule Mix.Tasks.Phx.Gen.Context do
     {opts, parsed, _} = parse_opts(args)
     [context_name, schema_name, plural | schema_args] = validate_args!(parsed)
 
-    table = Keyword.get(opts, :table, Phoenix.Naming.underscore(context_name) <> "_#{plural}")
+    table = Keyword.get(opts, :table, (context_name 
```

Comment:
Instead of inline such a huge expression, please move it a variable. Something like:

    table_prefix =  context_name |> Phoenix.Naming.underscore |> String.replace("/", "_")
    table = Keyword.get(opts, :table, "#{table_prefix}_#{plural}")